src/libs/pr.ts

```diff
@@ -1,9 +1,23 @@
 import * as core from '@actions/core';
 import { context, getOctokit } from '@actions/github';
-import * as dedent from 'dedent';
+import AnsiToHtml from 'ansi-to-html';
+import dedent from 'dedent';
 import invariant from 'ts-invariant';
 import { Config } from '../config';
 
+function ansiToHtml(
+  message: string,
+  { maxLength }: { maxLength?: number } = { maxLength: undefined },
+) {
+  const convert = new AnsiToHtml();
+  let html = convert.toHtml(message);
+  while (maxLength !== undefined && html.length > maxLength) {
+    message = message.substring(0, message.length - 1);
+    html = convert.toHtml(message);
+  }
```

Comment:
Could you explain in a comment what this code is doing? It looks like it's truncating the message body according to the max length of the message.

However, it looks like it's only cutting out a single character at a time and attempting to convert to HTML again. I imagine that would be very slow if html.length >>>> maxLength.